tests/test_specfile.py

```diff
@@ -13,6 +13,7 @@ def setUpClass(self):
     def setUp(self):
         url = "http://www.testpkg.com/testpkg/pkg-1.0.tar.gz"
         self.specfile = specfiles.Specfile(url, '1.0', 'pkg', '2')
+        specfiles.config.config_opts['dev_requires_extras'] = False
```

Comment:
This is fine, though it exposes similar timing problems because it doesn't follow a backup and restore once finished pattern. We don't do that hardly anywhere though so I think it is more symptomatic of the problems autospec has with global state mutation so more reasons to refactor.